pyproject.toml

```diff
@@ -55,6 +55,8 @@ sdist.cmake = true
 
 [tool.scikit-build.cmake.define]
 TILEDB_VS_PYTHON = "ON"
+INSTALL_LIB_TILEDB = {env="INSTALL_LIB_TILEDB"}
+TileDB_DIR = {env="TileDB_DIR"}
```

Comment:
Any particular motivation to use the env var name `TileDB_DIR`? I'm trying to reduce the number of variable names/patterns I have to remember across feedstocks.

* SOMA uses `TILEDB_PATH` and `TILEDBSOMA_PATH`
* VCF uses `LIBTILEDBVCF_PATH`

Could tiledb-vector-search also use `TILEDB_PATH` to match [SOMA](https://github.com/single-cell-data/TileDB-SOMA/blob/65e6bafa808d06ebdb5aa79d7f5dc83e8af422a8/apis/python/setup.py#L57)?

Reply:
To be honest, these values do not need to be in `pyproject.toml` so I'll remove them. They only need to be present in CMake because they are only used when building from `sdist`. I decided to divide this logic into two separate things since they are two separate things:

1. Pick and choose which tiledb core library is used
2. Decide whether to include or exclude tiledb core library in install group.

This way we have a fine grained control over the build process. However, I am all for the consistency so I'll rework this into using just `TILEDB_PATH` that would set both of those things at once.